<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shellOptions</class>
 <widget class="QWidget" name="shellOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>112</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" rowspan="5" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="gbAdvanced">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="3" column="1">
         <widget class="QLabel" name="lbUsername">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Enter the user you want to run the application as here.</string>
          </property>
          <property name="text">
           <string>User&amp;name:</string>
          </property>
          <property name="buddy">
           <cstring>leUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="KLineEdit" name="lePassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>5</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Enter the password here for the user you specified above.</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lbPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Enter the password here for the user you specified above.</string>
          </property>
          <property name="text">
           <string>Pass&amp;word:</string>
          </property>
          <property name="buddy">
           <cstring>lePassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QCheckBox" name="cbRunInTerminal">
          <property name="whatsThis">
           <string>Check this option if the application you want to run is a text mode application. The application will then be run in a terminal emulator window.</string>
          </property>
          <property name="text">
           <string>Run in &amp;terminal window</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="KLineEdit" name="leUsername">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>5</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Enter the user you want to run the application as here.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QCheckBox" name="cbRunAsOther">
          <property name="whatsThis">
           <string>Check this option if you want to run the application with a different user id. Every process has a user id associated with it. This id code determines file access and other permissions. The password of the user is required to do this.</string>
          </property>
          <property name="text">
           <string>Run as a different &amp;user</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbRunInTerminal</tabstop>
  <tabstop>cbRunAsOther</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbRunAsOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRunAsOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>leUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRunAsOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRunAsOther</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
